eeg_flow/oddball/oddball.py

```diff
@@ -0,0 +1,334 @@
+        if trial != "cross":
+            expected_idx += 1
+        lines_checked.append((idx, trial))
+    return lines_checked
+
+
+def _list_novel_sounds() -> List[str]:
+    """List the available sounds."""
+    novel_sounds = list()
+    for file in (files("eeg_flow.oddball") / "sounds").iterdir():
+        if file.suffix != ".wav":
+            logger.warning("Non-wav file %s found in the sounds directory.", file)
+            continue
+        if file.name.startswith("wav"):
+            novel_sounds.append(file.name)
+    return novel_sounds
+
+
+def _load_sounds(trials) -> Dict[str, SoundPTB]:
+    """Create psychopy sound objects."""
+    sounds = dict()
+    fname_standard = files("eeg_flow.oddball") / "sounds" / "low_tone-48000.wav"
+    fname_standard = ensure_path(fname_standard, must_exist=True)
+    sounds["standard"] = SoundPTB(
+        fname_standard, secs=_DURATION_STIM, hamming=True, name="stim", sampleRate=48000
+    )
+    fname_target = files("eeg_flow.oddball") / "sounds" / "high_tone-48000.wav"
+    fname_target = ensure_path(fname_target, must_exist=True)
+    sounds["target"] = SoundPTB(
+        fname_target, secs=_DURATION_STIM, hamming=True, name="stim", sampleRate=48000
+    )
+
+    novels = [trial[1] for trial in trials if trial[1].startswith("wav")]
+    for novel in novels:
+        fname = files("eeg_flow.oddball") / "sounds" / f"{novel}-48000.wav"
+        sounds[novel] = SoundPTB(
+            fname, secs=_DURATION_STIM, hamming=True, name="stim", sampleRate=48000
+        )
+    return sounds
+
+
+def _load_cross(
+    win: Window, passive: bool
+) -> Dict[str, Union[ShapeStim, Tuple[ShapeStim, ShapeStim]]]:
+    """Load the shapes used for fixation cross.
+
+    The point position is defined as:
+
+            0  11
+
+
+
+     2      1  10     9
+
+     3      4  7      8
+
+
+
+            5  6
+    """
+    check_type(passive, (bool,), "passive")
+    crosses = dict()
+    # convert the number of pixels into the normalized unit per axis (x, y)
+    width = _CROSS_WIDTH * 2 / win.size
+    length = _CROSS_LENGTH * 2 / win.size
+    points = np.array(
+        [
+            [-width[0] / 2, width[1] / 2 + length[1]],  # top-left
+            [-width[0] / 2, width[1] / 2],  # center-left-up
+            [-width[0] / 2 - length[0], width[1] / 2],  # left-up
+            [-width[0] / 2 - length[0], -width[1] / 2],  #  left-bottom
+            [-width[0] / 2, -width[1] / 2],  # center-left-bottom
+            [-width[0] / 2, -width[1] / 2 - length[1]],  # bottom-left
+            [width[0] / 2, -width[1] / 2 - length[1]],  # bottom-right
+            [width[0] / 2, -width[1] / 2],  # center-right-bottom
+            [width[0] + length[0], -width[1] / 2],  # right-bottom
+            [width[0] + length[0], width[1] / 2],  # right-up
+            [width[0] / 2, width[1] / 2],  # center-right-up
+            [width[0] / 2, width[1] / 2 + length[1]],  # top-right
+        ]
+    )
```

Comment:
Definition of the cross vertices, maybe I made a mistake using the wrong indices [1] vs [0] for the right arm.